Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM  ubuntu:16.04 as builder
+
+USER  root
+
+ENV OPT /opt/wtsi-cgp
+ENV PATH $OPT/bin:$PATH
+ENV LD_LIBRARY_PATH $OPT/lib
+
+RUN apt-get -yq update
+RUN apt-get install -yq --no-install-recommends\
+  locales\
+  libperlio-gzip-perl \
+  cpanminus \
+  build-essential\
+  apt-transport-https\
+  curl\
+  ca-certificates\
+  make\
+  gcc
```

Comment:
safe to remove the two. seems installation of cpanm will have them (or parts of them) installed anyway.